### ISSUE TITLE
Why does the Albumentations keep turning on during training?

### ISSUE THREAD

**Issue by noorchauhan** [ultralytics/ultralytics#21275]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I keep my augmentations _off_ and still I see **albumentations** on while training, I tried to read the documentations again, but still it does not help.

Below is a screenshot attached:

![Image](https://github.com/user-attachments/assets/a1ddeda0-764a-4d83-94a1-bcfb43837ba9)

Guide to a solution if this is pointed out in any other issue, I have read the documentations yet no solutions 

Below is my code snippet for more information:

```python results = model.train( data="/kaggle/input/1by45c-config-file/config.yaml", device=[0,1,2,3], imgsz=1920, epochs=500, batch=8, patience=5, name="upto_2_may", optimizer='auto', cache=False, hsv_h=0.0, hsv_s=0.0, hsv_v=0.0, degrees=0.0, translate=0.0, scale=0.0, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.0, bgr=0.0, mosaic=0.0, mixup=0.0, cutmix=0.0, copy_paste=0.0, erasing=0.0, auto_augment=None ) ```

**Comment by UltralyticsAssistant** [comment]
👋 Hello @noorchauhan, thank you for reaching out and sharing your question with us! 🚀 This is an automated response to help guide you while an Ultralytics engineer reviews your issue.

For new users, we recommend visiting the [Docs](https://docs.ultralytics.com/) where you’ll find a variety of [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) examples. Many common questions are answered there, so it's a great resource to explore.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) (MRE) if possible to help us debug further.

If your inquiry is about custom training or configuration, please include as much detail as you can—such as dataset image examples, your config files, and full training logs. Also, make sure to review our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

You can connect with the Ultralytics community for more support:
- For real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
- For in-depth discussions: [Discourse](https://community.ultralytics.com/)
- Or join threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share and learn with others.

## Upgrade

Please ensure you are running the latest ultralytics package and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). This helps confirm your issue isn't already fixed in the latest release:

```bash
pip install -U ultralytics
```

## Environments

YOLO models can be run in any of the following up-to-date verified environments (with all dependencies like [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. These tests verify all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

An Ultralytics engineer will review your issue soon and provide further assistance. Thank you for your patience and for being part of the Ultralytics community! 🏆

**Comment by Y-T-G** [comment]
To disable albumentations, you need to uninstall it.

**Comment by jugal-sheth** [comment]
You can also try `auto_augment=False`

**Comment by noorchauhan** [comment]
This issue was resolved by uninstalling albumentations.

```
pip uninstall albumentations -y
```

**Comment by glenn-jocher** [comment]
That's correct! The [Albumentations integration](https://docs.ultralytics.com/reference/data/augment/#ultralytics.data.augment.Albumentations) is automatically included in the training pipeline when the package is installed, even with default augmentation parameters set to 0.0. The `Albumentations(p=1.0)` transform is always added to the composition, but when the package isn't installed, it simply passes through without applying any transformations. Uninstalling albumentations is indeed the proper solution to completely disable it from appearing in your training logs.
